pkg/apis/tenancy/v1alpha1/helper/helper_test.go

```diff
@@ -136,25 +138,44 @@ func TestParseLogicalClusterName(t *testing.T) {
 	}
 }
 
-func TestWorkspaceKey(t *testing.T) {
+func TestHasClusterWorkspace(t *testing.T) {
 	tests := []struct {
 		name string
 		org  string
 		ws   string
-		want string
+		want bool
+	}{{
+		name: "root ws",
+		org:  "",
+		ws:   RootCluster,
+		want: false,
+	}, {
+		name: "system:admin ws",
+		org:  "system:admin",
```

Comment:
Why do we supply `ws: "root"` but `org: "system:admin` ?

Reply:
I mean to say, the root cluster is represented by `org: "", ws: "root"`, whereas the system:admin cluster is represented by `org: "system:admin", ws: "<arbitrary>"`. I inherited this, I'd like to understand why it makes sense.

Reply:
that's probably from a blind rewrite from our former admin:foo. We basically have split former admin into system:admin and root. So +1 for removing the case. system:admin should never be in org.

Reply:
+1

Reply:
I've update the code to check for `:` and updated this test case accordingly.

Reply:
Done.